### PR TITLE
Make gem work for projects with Ruby >= 2.7 and source code with UTF-8

### PR DESCRIPTION
**Pull request by jmgarnier**
It sounds like this projet is kind of abandoned so I'll push this PR for people who have read your excellent blog post and want to give it a try.

Thank you for sharing :)

### CODE REVIEW

**Review by DanielePalombo — APPROVED** [review]
LGTM

File: lib/reverse_coverage/formatters/html/formatter.rb [hunk 1]
```diff
@@ -58,21 +58,53 @@ def formatted_source_file(source_file)
...
             'Channels' => %r{/app/channels},
             'Models' => %r{/app/models},
             'Mailers' => %r{/app/mailers},
             'Helpers' => %r{/app/helpers},
             'Jobs' => %r{/app/jobs|/app/workers},
-            'Libraries' => %r{/lib/}
+            'Libraries' => %r{/lib/},
+            'Interactors' => %r{/app/interactors},
+            'Services' => %r{/app/services},
+            'Validators' => %r{/app/validators},
```
Comment (jmgarnier) [inline lib/reverse_coverage/formatters/html/formatter.rb:107]: should probably removes these 3 last lines. Not standard

Comment (DanielePalombo) [reply lib/reverse_coverage/formatters/html/formatter.rb:107]: Yes, maybe you can add a configuration in order to customize the Categories

Comment (jmgarnier) [reply lib/reverse_coverage/formatters/html/formatter.rb:107]: It's been 2 years. I won't work on it. You can remove these lines with a new commit

Comment (DanielePalombo) [reply lib/reverse_coverage/formatters/html/formatter.rb:107]: I cannot edit your branch. I would prefer it if you can remove that commit and force-push the change.

**Review by DanielePalombo — CHANGES_REQUESTED** [review]

**Review by DanielePalombo — APPROVED** [review]

**Comment by jmgarnier** [comment]
❤️

**Pull request merged**